src/Features/LanguageServer/Protocol/Handler/Formatting/AbstractFormatDocumentHandlerBase.cs

```diff
@@ -30,7 +30,7 @@ internal abstract class AbstractFormatDocumentHandlerBase<RequestType, ResponseT
 
             if (document != null)
             {
-                var formattingService = document.Project.LanguageServices.GetRequiredService<IEditorFormattingService>();
+                var formattingService = document.Project.LanguageServices.GetRequiredService<IFormattingInteractionService>();
```

Comment:
This doesn't need to use the proxy?

Reply:
Based on https://github.com/dotnet/roslyn/pull/50222#issuecomment-845329972, no.